source/agora/consensus/EnrollmentManager.d

```diff
@@ -1256,3 +1291,72 @@ unittest
     man.getEnrollments(enrolls, Height(validator_cycle + 11));
     assert(enrolls.length == 0);
 }
+
+// test getRandomSeed()
+unittest
+{
+    import agora.common.crypto.Schnorr;
+    import agora.consensus.data.Transaction;
+    import std.conv;
+    import std.range;
+    import std.stdio;
+
+    scope storage = new TestUTXOSet;
+    Hash[] utxos;
+
+    // genesisSpendable returns 8 outputs
+    auto pairs = iota(8).map!(idx => WK.Keys[idx]).array;
```

Comment:
We should really use `repeat` more often :)

Reply:
Also here you could also write `WK.Keys.byRange().take(8).array`.

Reply:
Oh neato. I forgot you added that.